### PR TITLE
Blockchain changes

### PR DESCRIPTION
**Pull request by Ronserruya**
* Main purpose:
Change kin-base due to blockchain changes.
 
* Technical description:

- Replace xlm with kin
- kin = 1e4 stroops instead 1e7
- fee is now verified to be an integer
- updated all the test environment

* Tests
Currently there is no 'master' version of the new blockchain changes, to I compiled horizon and stellar-core locally to test and the tests are passing on my pc. Whenever a master version will be released I will edit the tests so they can work on travis

### CODE REVIEW

File: images/docker-compose.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ services:
       POSTGRES_DB: core
 
   horizon:
-    image: kinecosystem/horizon:v0.12.3-sse-pubsub-performance-improvement
+    image: kinecosystem/horizon:ronlocal
```
Comment (doodyparizada) [inline images/docker-compose.yml:52]: maybe change the name here?

File: kin_base/operation.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
                     is_valid_address, convert_hex_to_bytes)
 from .exceptions import StellarAddressInvalidError, NotValidParamError
 
-ONE = Decimal(10 ** 7)
+ONE = Decimal(10 ** 4)  # TODO: move to a config file
```
Comment (doodyparizada) [inline kin_base/operation.py:14]: Not sure it should be a config. It will never change

File: kin_base/transaction.py [hunk 3]
```diff
@@ -68,6 +68,9 @@ def __init__(self,
         self.source = source
         self.sequence = int(sequence) + 1
         self.memo = memo or NoneMemo()
+        if fee is not None:
```
Comment (doodyparizada) [inline kin_base/transaction.py:71]: use isinstance to check if the type is int

**Review by doodyparizada — COMMENTED** [review]
some comments

**Review by ravitb — APPROVED** [review]

**Pull request merged**